eclipse/sw360
# feat(ProjectUI): Add new values to Obligation status

[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.

Updated the obligation status as follows:
* **Mapped the following old status to new status:**
_Old status_ -> _New status_ 
`Fulfilled` -> `Acknowledged or fulfilled` 
`In progress` -> `Will be fulfilled before release`
`To be fulfilled by parent project` -> `Deferred to parent product`

* **Added following new status:**
```
Fulfilled and parent must also fulfill
Escalated
```

* **Retained old status as is, as there was no change:**
```
Open 
Not applicable
```

> Which issue is this pull request belonging to and how is it solving it?
* #1351 

> Did you add or update any new dependencies that are required for your change?
* No

Issue: #1351 

### Suggest Reviewer
> You can suggest reviewers here with an @mention.

### How To Test?
* Run the migration script `047_migrate_obligation_status.py`, to change the old status into new status. (python 047_migrate_obligation_status.py 2>&1 | tee -a output.log)
* User should be able to update the Linked obligation status, and add comments to obligations.

### Checklist
Must:

Signed-off-by: Abdul Kapti <abdul.kapti@siemens-healthineers.com>
